src/conf_mode/service_snmp.py

```diff
@@ -85,8 +86,18 @@ def get_config(config=None):
             tmp = {'::1': {'port': '161'}}
             snmp['listen_address'] = dict_merge(tmp, snmp['listen_address'])
 
+    if 'script_extensions' in snmp:
+        for key, val in snmp['script_extensions'].get('extension_name').items():
+            script_path = val.get('script')
+            # if script has not absolute path, use pre configured path
+            if not os.path.isabs(script_path):
+                script_path = os.path.join(default_script_dir, script_path)
```

Comment:
Only using `set service snmp script-extensions extension-name bar` results in:

```
Traceback (most recent call last):
  File "/usr/libexec/vyos/conf_mode/service_snmp.py", line 272, in <module>
    c = get_config()
        ^^^^^^^^^^^^
  File "/usr/libexec/vyos/conf_mode/service_snmp.py", line 92, in get_config
    if not os.path.isabs(script_path):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 62, in isabs
TypeError: expected str, bytes or os.PathLike object, not NoneType
```

Please us this instead:

```python3
    if 'script_extensions' in snmp and 'extension_name' in snmp['script_extensions']:
        for key, val in snmp['script_extensions']['extension_name'].items():
            if 'script' not in val:
                continue
            script_path = val['script']
            # if script has not absolute path, use pre configured path
            if not os.path.isabs(script_path):
                script_path = os.path.join(default_script_dir, script_path)

            snmp['script_extensions']['extension_name'][key]['script'] = script_path

```

Reply:
fixed